### ISSUE TITLE
typo when setting SO_SNDBUF?

### ISSUE THREAD

**Issue by Jamol** [facebook/folly#1316]
in AsyncUDPSocket::bind(const folly::SocketAddress& address):
```
if (sndBuf_ > 0) {
    // Set the size of the buffer for the sent messages in tx_queues.
    **int value = rcvBuf_;**
    if (netops::setsockopt(
            socket, SOL_SOCKET, SO_SNDBUF, &value, sizeof(value)) != 0) {
      throw AsyncSocketException(
          AsyncSocketException::NOT_OPEN,
          "failed to set SO_SNDBUF on the socket",
          errno);
    }
  }
```
should "int value = rcvBuf_; "  be "int value = sndBuf_;" ?

**Comment by afrind** [comment]
This looks like a bug.  Would you like to submit a PR to fix?

**Comment by yfeldblum** [comment]
Was this fixed?

**Comment by afrind** [comment]
fixed by #1319
